tests/playwright/src/model/pages/kubernetes-dashboard-page.ts

```
@@ -48,7 +69,35 @@ export class KubernetesDashboardPage extends MainPage {
+  }
+
+  async getCurrentActiveCountForResource(name: KubernetesResources): Promise<number> {
+    const currentResource = this.metricsLocatorByName(name);
+    const activeCountLocator = currentResource.getByLabel(`${name} active count`, { exact: true });
+
+    if ((await activeCountLocator.count()) === 0) {
+      throw new Error(`No active count locator found for resource: ${name}`);
+    }
+
+    const activeCountText = await activeCountLocator.textContent();
+    return activeCountText ? parseInt(activeCountText, 10) : 0;
```

Biome (2.1.2)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)